### PR TITLE
add a USAGE document

### PR DESCRIPTION
**Pull request by gregwebs**

### CODE REVIEW

File: USAGE.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+## Overriding the aws CLI to use aws-vault
+
+You can create an overriding script (make it higher precedence in your PATH) that looks like the below:
+
+```
+#!/bin/bash
+set -euo pipefail
+ 
+AWS_PROFILE=${AWS_DEFAULT_PROFILE:-work}
```
Comment (bradfeehan) [inline USAGE.md]: You should add quotes to support spaces in the profile name:

```
AWS_PROFILE="${AWS_DEFAULT_PROFILE:-work}"
```

File: USAGE.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+exec aws-vault exec "$AWS_PROFILE" -- /usr/local/bin/aws "$@"
```
Comment (bradfeehan) [inline USAGE.md]: Maybe bail early if `AWS_VAULT` is already set?

```bash
if [[ "${AWS_VAULT:-}" = "$AWS_PROFILE" ]]; then
  # desired profile already active
  exec /usr/local/bin/aws "$@"
elif [[ "$AWS_VAULT" ]]; then
  echo "Error activating aws-vault profile '$AWS_PROFILE'"
  echo "A different profile is already active: '$AWS_VAULT'"
  exit 1
fi
```

File: README.md [hunk 3]
```diff
@@ -37,17 +39,8 @@ $ aws-vault exec home -- aws s3 ls
...
-Enter Secret Key: %
-
-# Execute a command using temporary credentials
-$ aws-vault exec work -- aws s3 ls
-another_bucket
-
 # Inspect the environment
-$ aws-vault exec work -- env | grep AWS
+$ aws-vault exec home -- env | grep AWS
 AWS_VAULT=work
```
Comment (bradfeehan) [inline README.md]: This should probably match the previous line

File: USAGE.md (hunk 2 above)
Comment (lox) [inline USAGE.md]: aws-vault will do that on it's own @bradfeehan

File: README.md (hunk 3 above)
Comment (gregwebs) [inline README.md]: Hopefully if you look at the new version it makes sense. Basically I am trying to tighten up the README here by not explicitly showing off multiple profiles.

**Comment by lox** [comment]
Thanks, this is a big improvement.

**Pull request merged**